src/Okta.Idx.Sdk.E2ETests/PageObjectModels/SelectAuthenticatorAsyncPage.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Okta.Idx.Sdk.E2ETests.Drivers;
+using OpenQA.Selenium;
+
+namespace Okta.Idx.Sdk.E2ETests.PageObjectModels
+{
+    public class SelectAuthenticatorAsyncPage : BasePage
```

Comment:
What's the difference between `SelectAuthenticatorAsyncPage` and `SelectAuthenticatorPage`?

Reply:
this is the same page with the same controls, but with different URLs, it can be handled in the same place, say 
`SelectAuthenticatorPage`, I think I will move all the code to the one class